examples/ipex_bert.yaml

```diff
@@ -25,5 +25,4 @@ backend:
   device: cpu
   no_weights: false
   export: true
-  torch_dtype: bfloat16
```

Comment:
removed because `BF16 weight prepack needs the cpu support avx_ne_convert or avx512bw, avx512vl and avx512dg`. We are running the test on `ubuntu-latest`. This must be some old CPU so it is not the best to test ipex IMO. Do you have an idea or preference for a more modern intel CPU to use for the CI?

Reply:
I prefer testing on a more advanced cpu machine, but minimal in terms of size / vCPUs.